docs/sources/tempo/operations/dedicated_columns.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Dedicated columns
+description: Learn how to use dedicated columns to improve query performance.
+weight: 42
+---
+
+# Dedicated columns
+
+Dedicated columns are a way to improve query performance by storing the most frequently used columns in dedicated columns,
+rather than in the generic attribute key-value list.
+
+This is a feature introduced with `vParquet3`, and is only available when using this Parquet storage format.
+To read more about the design of `vParquet3`, see [the design proposal](https://github.com/grafana/tempo/blob/main/docs/design-proposals/2023-05%20vParquet3.md).
+
+## Configuration
+
+Dedicated columns can be configured in the storage block or via overrides.
+
+```yaml
+# Storage configuration for traces
+storage:
+  trace:
+    block:
+      # Default dedicated columns for all blocks
+      dedicated_columns:
+        - name: <string>, # name of the attribute
+          type: <string>, # type of the attribute. options: string
+          scope: <string> # scope of the attribute. options: resource, span
+
+overrides:
+  # Global overrides for dedicated columns configuration
+  dedicated_columns:
+    - name: <string>, # name of the attribute
+      type: <string>, # type of the attribute. options: string
+      scope: <string> # scope of the attribute. options: resource, span
+
+  per_tenant_override_config: /conf/overrides.yaml
+---
+# /conf/overrides.yaml
+# Tenant-specific overrides configuration
+overrides:
+  "<tenant id>":
+    dedicated_columns:
+      - name: <string>, # name of the attribute
+        type: <string>, # type of the attribute. options: string
+        scope: <string> # scope of the attribute. options: resource, span
+
+  # A "wildcard" override can be used that will apply to all tenants if a match is not found otherwise.
```

Comment:
"Otherwise" was redundant since you also say "if a match is not found".
```suggestion
  # A "wildcard" override can be used that will apply to all tenants if a match is not found.
```